api/config/cdr_config_preprod.json

```diff
@@ -70,15 +70,15 @@
     },
     {
       "cdrVersionId": 5,
-      "name": "All of Us Controlled Tier Dataset v5 alpha 1",
+      "name": "All of Us Controlled Tier Dataset v5 alpha 2",
```

Comment:
I was considering dropping the number from this.. but I think I actually prefer updating it like you've done here so the user has some confirmation.